flexmeasures_entsoe/utils.py

```diff
@@ -157,53 +157,58 @@ def abort_if_data_empty(data: Union[pd.DataFrame, pd.Series]):
         raise click.Abort
 
 
-def parse_from_and_to_dates_default_today_and_tomorrow(
-    from_date: Optional[datetime], to_date: Optional[datetime], country_timezone: str
-) -> Tuple[datetime, datetime]:
-    """
-    Parse CLI options (or set default to today and tomorrow)
-    Note:  entsoe-py expects time params as pd.Timestamp
-    """
-    today_start = datetime.today().replace(hour=0, minute=0, second=0, microsecond=0)
-    if to_date is None:
-        to_date = pd.Timestamp(
-            today_start, tzinfo=pytz.timezone(country_timezone)
-        ) + pd.offsets.DateOffset(
-            days=1
-        )  # Add a calendar day instead of just 24 hours, from https://github.com/gweis/isodate/pull/64
-    else:
-        to_date = pd.Timestamp(to_date, tzinfo=pytz.timezone(country_timezone))
-    if from_date is None:
-        from_date = pd.Timestamp(today_start, tzinfo=pytz.timezone(country_timezone))
-    else:
-        from_date = pd.Timestamp(from_date, tzinfo=pytz.timezone(country_timezone))
-    from_time, until_time = date_range_to_time_range(from_date, to_date)
-    return from_time, until_time
+def abort_if_data_incomplete(
+    data: Union[pd.DataFrame, pd.Series],
+    from_time: pd.Timestamp,
+    until_time: pd.Timestamp,
+    resolution: pd.Timedelta,
+):
+    expected_periods = int((until_time - from_time) / resolution)
+    if len(data) < expected_periods:
+        click.secho(
+            f"Result is incomplete. Expected {expected_periods} periods but got {len(data)}. Probably ENTSO-E does not provide these forecasts yet ...",
+            **MsgStyle.ERROR,
+        )
+        raise click.Abort
 
 
-def parse_from_and_to_dates_default_yesterday(
-    from_date: Optional[datetime], to_date: Optional[datetime], country_timezone: str
+def parse_from_and_to_dates(
+    from_date: Optional[datetime],
+    until_date: Optional[datetime],
+    country_timezone: str,
+    default_to="today-and-tomorrow",  # Can be "tomorrow" or "today"
 ) -> Tuple[datetime, datetime]:
     """
-    Parse CLI options (or set default to yesterday)
+    Parse CLI options for start and end date (or set default to today and tomorrow) for inout to entsoe-py
+    Note: we expect only dates as input here, and until_date is inclusice, so we extend it with 24h - so if from_date is equal to until_date, we return 00:00 and 24:00 of that day.
     Note:  entsoe-py expects time params as pd.Timestamp
     """
+    tz = pytz.timezone(country_timezone)
+    now = datetime.now(tz)
+    today_start = now.replace(hour=0, minute=0, second=0, microsecond=0)
+
+    if default_to == "tomorrow":
+        default_start = today_start + pd.DateOffset(days=1)
```

Comment:
```suggestion
        default_start = today_start + timedelta(days=1)
```

This is just Python code, we can use standard Python library things. Also in the others.